api/tests/test_asset_api.py

```diff
@@ -387,3 +387,63 @@ def test_asset_allocation_history_has_assigner(self, mock_verify_id_token):
         )
         self.assertIn("assigner", response.data["allocation_history"][0].keys())
         self.assertEqual(response.status_code, 200)
+
+    @patch("api.authentication.auth.verify_id_token")
+    def test_asset_search_endpoint(self, mock_verify_id_token):
```

Comment:
Asset search is more of a functionality than an endpoint. However, there's an asset endpoint. A better description could be, "test_search_funx_on_asset_endpoint".